pkg/vault/contracts/RouterHooks.sol

```diff
@@ -25,13 +25,22 @@ contract RouterHooks is RouterCommon {
     using RouterWethLib for IWETH;
     using SafeCast for *;
 
+    /**
```

Comment:
Not in diff, but imports should be as below. Also "Base Router with hooks" reads better as "Base router contract with hooks..."

```
import { SafeCast } from "@openzeppelin/contracts/utils/math/SafeCast.sol";
import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
import { IPermit2 } from "permit2/src/interfaces/IPermit2.sol";

import { IWETH } from "@balancer-labs/v3-interfaces/contracts/solidity-utils/misc/IWETH.sol";
import { IVault } from "@balancer-labs/v3-interfaces/contracts/vault/IVault.sol";
import "@balancer-labs/v3-interfaces/contracts/vault/RouterTypes.sol";

import { RouterWethLib } from "./lib/RouterWethLib.sol";
import { RouterCommon } from "./RouterCommon.sol";
```